packages/firebase_crashlytics/firebase_crashlytics_platform_interface/lib/src/method_channel/method_channel_crashlytics.dart

```diff
@@ -71,32 +70,32 @@ class MethodChannelFirebaseCrashlytics extends FirebaseCrashlyticsPlatform {
   }
 
   @override
-  Future<bool /*!*/ > didCrashOnPreviousExecution() async {
+  Future<bool> didCrashOnPreviousExecution() async {
     try {
-      Map<String, dynamic> data =
+      Map<String, dynamic>? data =
           await channel.invokeMapMethod<String, dynamic>(
               'Crashlytics#didCrashOnPreviousExecution');
 
-      return data['didCrashOnPreviousExecution'];
+      return data!['didCrashOnPreviousExecution'];
     } catch (e, s) {
       throw convertPlatformException(e, s);
     }
   }
 
   @override
   Future<void> recordError({
-    String /*!*/ exception,
-    String /*!*/ information,
-    String /*?*/ reason,
-    List<Map<String, String>> /*?*/ stackTraceElements,
```

Comment:
This was marked with `?`. Why use `required`?

Maybe the prototype of `recordError` is invalid? 🤔

Reply:
If I remember correctly, I updated it because the user facing API will always pass a stack trace. Happy to change it back though as I guess the default makes sense.